examples/classification_3d/densenet_training_array.py

```diff
@@ -27,27 +28,28 @@ def main():
 
     # IXI dataset as a demo, downloadable from https://brain-development.org/ixi-dataset/
     images = [
-        "/workspace/data/medical/ixi/IXI-T1/IXI314-IOP-0889-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI249-Guys-1072-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI609-HH-2600-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI173-HH-1590-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI020-Guys-0700-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI342-Guys-0909-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI134-Guys-0780-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI577-HH-2661-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI066-Guys-0731-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI130-HH-1528-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI607-Guys-1097-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI175-HH-1570-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI385-HH-2078-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI344-Guys-0905-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI409-Guys-0960-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI584-Guys-1129-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI253-HH-1694-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI092-HH-1436-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI574-IOP-1156-T1.nii.gz",
-        "/workspace/data/medical/ixi/IXI-T1/IXI585-Guys-1130-T1.nii.gz",
+        os.sep.join(["workspace", "data", "medical", "ixi", "IXI-T1", "IXI249-Guys-1072-T1.nii.gz"]),
```

Comment:
May I know why you change the images in the list?
Thanks.

Reply:
There was no reason apart from confusion, I have checked the order again and restablish as it was before. My apologies.